RequestService/RequestService.Repo/Repository.cs

```diff
@@ -2155,5 +2156,189 @@ public async Task<Chart> GetActivitiesByMonth(int groupId)
             
             return result;
         }
+
+        public async Task<Chart> RequestVolumeByDueDateAndRecentStatus(int groupId)
+        {
+            DateTime dt = DateTime.UtcNow.Date.AddMonths(-13);
+            Chart result = new Chart()
+            {
+                Title = "Request volume by due date and most recent status",
+                XAxisName = "Month",
+                YAxisName = "Count",
+                ChartType = ChartTypes.Bar,
+                ChartItems = new List<ChartItem>()
+            };
+
+            var chartItems = _context.Job
+                .Include(i => i.NewRequest)
+                .Where(x => x.NewRequest.ReferringGroupId == groupId && x.DueDate > dt)
+                .GroupBy(g => new { g.JobStatusId, date = g.DueDate })
```

Comment:
Looks like we're grouping by date here, then secondarily by month below.

I'm concerned that if there are two jobs due at exactly the same time, they will be grouped here as one, with Count = 2, but that count will then be ignored and they will be counted as a single item on line 2222.

Reply:
(Same applies to other charts.)